src/hyperion/experiment_plans/tests/test_flyscan_xray_centre_plan.py

```diff
@@ -329,53 +329,17 @@ def test_individual_plans_triggered_once_and_only_once_in_composite_run(
     move_xyz.assert_called_once()
 
 
-@patch(
-    "dodal.devices.aperturescatterguard.ApertureScatterguard._safe_move_within_datacollection_range",
-    autospec=True,
-)
 @patch("hyperion.experiment_plans.flyscan_xray_centre_plan.run_gridscan", autospec=True)
 @patch("hyperion.experiment_plans.flyscan_xray_centre_plan.move_x_y_z", autospec=True)
-@patch("bluesky.plan_stubs.rd")
-def test_logging_within_plan(
-    rd: MagicMock,
+def test_when_gridscan_finished_then_smargon_stub_offsets_are_set(
```

Comment:
Did you get rid of this test for a reason?

Reply:
It's hard to see here because GH has messed up the diff but if you look on main you can see that this test and the one above it were duplicates